core/dbt/main.py

```diff
@@ -615,17 +615,10 @@ def _add_common_selector_arguments(sub):
     )
 
 
-def _add_selection_arguments(*subparsers, **kwargs):
-    models_name = kwargs.get('models_name', 'models')
+def _add_selection_arguments(*subparsers):
     for sub in subparsers:
-        if models_name == 'models':
-            _add_models_argument(sub)
-        elif models_name == 'select':
-            # these still get stored in 'models', so they present the same
-            # interface to the task
```

Comment:
Could we change this, from both storing in `args.models` to both storing in `args.select`? To my mind, this is most relevant for:
- task code (e.g. [compile](https://github.com/dbt-labs/dbt/blob/7fa14b6948d48bfaf521a2a24587e2ec79e1e89a/core/dbt/task/compile.py#L44))
- the `args` dict that gets stored in `run_results.json`, which right now is always called `models` (for both `--models` and `--select`)

Since we're going to start documenting `select` across the board, and only keep `--models` support for backwards compatibility, I'd prefer that the other interfaces reflect the `select` nomenclature. But I'd want to know how big a lift that would be.

Reply:
I'll take a look at what changes would be involved.

Reply:
@gshank @jtcohen6 if this PR is good, lets merge this in and then tackle the possible additional work in a new PR

Reply:
Oops. Sorry. Didn't see Leah's comment and the extra work is done.